tutorials/javaFxPart1.md

```diff
@@ -94,6 +94,18 @@ public class Launcher {
 }
 ```
 
+Now that we have changed the entry point to our application, we need to update the `mainClass`attribute in `build.gradle` accordingly to point at the `Launcher` class.
+
+```groovy{highlight-lines="3-5"}
+// ...
+
+application {
+   mainClass.set("Launcher")
```

Comment:
```suggestion
   // Launcher initializes JavaFX application and handles any pre-launch setup
   mainClass.set("Launcher")
```

This small change might provide context about the purpose of the Launcher class, making it clearer why it's being set as the main class. It helps future developers quickly understand the role of this class in the application structure without having to look up the Launcher class directly.

Reply:
Thanks for the suggestion, @baskargopinath! Although this might be useful for the user, I don't think that there is a need to emphasise this here, as the user is already aware that `Launcher` is a class which acts as a workaround which allows JavaFX to be integrated into the project and handles initialisation in the section directly before this one.

<img width="1015" alt="Screenshot 2024-08-01 at 1 04 16 PM" src="https://github.com/user-attachments/assets/f7d00085-d341-4dba-9a16-ee27b12d49ac">

Perhaps this might warrant a new issue being made, as I think that the point of this PR is to remedy the issue of students not knowing how to implement this workaround if they have constructors with arguments.